pkg/models/resources/v1alpha3/resource/resource.go

```diff
@@ -113,6 +115,7 @@ func NewResourceGetter(factory informers.InformerFactory, cache cache.Cache) *Re
 	namespacedResourceGetters[schema.GroupVersionResource{Group: "batch", Version: "v1", Resource: "jobs"}] = job.New(factory.KubernetesSharedInformerFactory())
 	namespacedResourceGetters[schema.GroupVersionResource{Group: "app.k8s.io", Version: "v1beta1", Resource: "applications"}] = application.New(cache)
 	namespacedResourceGetters[schema.GroupVersionResource{Group: "", Version: "v1", Resource: "persistentvolumeclaims"}] = persistentvolumeclaim.New(factory.KubernetesSharedInformerFactory(), factory.SnapshotSharedInformerFactory())
+	namespacedResourceGetters[schema.GroupVersionResource{Group: "", Version: "v1", Resource: "persistentvolumes"}] = persistentvolume.New(factory.KubernetesSharedInformerFactory())
```

Comment:
The `persistentVolumeGetter` should be placed under the `clusterResourceGetters`, since PVs are cluster scoped, although it can also be retrieved now, because of this catch-all logic: https://github.com/kubesphere/kubesphere/blob/51955831a774d4d2b9132b638c9d1c64ac483f33/pkg/models/resources/v1alpha3/resource/resource.go#L162-L174
It's semantically correct and more efficient to retrieve when it's amended.